testprojects/elasticsearch/src/test/kotlin/io/cloudflight/testresources/springboot/elasticsearch/ApplicationTest.kt

```diff
@@ -0,0 +1,19 @@
+package io.cloudflight.testresources.springboot.elasticsearch
+
+import co.elastic.clients.elasticsearch.ElasticsearchClient
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+
+@SpringBootTest
```

Comment:
imho, we should test that here with the AutoConfiguration classes of SpringBoot for ElasticSearch and check that all properties are being bound correctly, rather than using the native ElasticsearchClient